QuizApp/Application/SceneDelegate.swift

```diff
@@ -15,8 +15,15 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
 
         guard let window = window else { return }
 
-        let loginViewModel = LoginViewModel()
-        window.rootViewController = LoginViewController(viewModel: loginViewModel)
+        let mainNavigationController = UINavigationController()
+        window.rootViewController = mainNavigationController
+
+        let dependenciesContainer: ContainerProvider = Container()
+        let appCoordinator: AppCoordinatorProvider = AppCoordinator(
```

Comment:
Please assign the coordinator to some class variable so that it is not destroyed (although it never will be because of reference with NavCon, but still).